components/object-property-list/object-property-list.js

```diff
@@ -41,6 +38,12 @@ class ObjectPropertyList extends LocalizeCoreElement(SkeletonMixin(LitElement))
 		`];
 	}
 
+	firstUpdated() {
+		this._slot = this.shadowRoot.querySelector('slot:not([name])');
+		this.addEventListener('d2l-object-property-list-item-visibility-change', this._onItemsChanged);
+		if (this._slot.childElementCount) this._onItemsChanged();
```

Comment:
Interesting... I never knew the slot had this property. Seems [Mozilla isn't forthcoming in these dets](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot).

Reply:
Though, are you sure this call is needed in `firstUpdated`? I think the `slotchange` event will be dispatched when the nodes are first distributed to it.

Reply:
This is specifically for the skeleton scenario, when nothing is slotted. `childElementCount` is to detect if there is any default content (e.g. skeleton) being rendered. So it's not anything specific to slots, rather just a default element attribute:
https://developer.mozilla.org/en-US/docs/Web/API/Element/childElementCount

In my testing this was necessary, otherwise the visual-diff for skeleton was rendering a separator on the final item.